pkg/client/informers/externalversions/factory.go

```diff
@@ -126,7 +126,7 @@ func (f *sharedInformerFactory) WaitForCacheSync(stopCh <-chan struct{}) map[ref
 	return res
 }
 
-// InternalInformerFor returns the SharedIndexInformer for obj using an internal
+// InformerFor returns the SharedIndexInformer for obj using an internal
```

Comment:
This is a generated file, as GitHub says, as is written at the top of the file, and per #12773

Reply:
Thanks. Reverted